src/vs/workbench/contrib/search/browser/anythingQuickAccess.ts

```diff
@@ -488,7 +488,10 @@ export class AnythingQuickAccessProvider extends PickerQuickAccessProvider<IAnyt
 
 	private createFileQueryCache(): FileQueryCacheState {
 		return new FileQueryCacheState(
-			cacheKey => this.fileQueryBuilder.file(this.contextService.getWorkspace().folders, this.getFileQueryOptions({ cacheKey })),
+			cacheKey => this.fileQueryBuilder.file(this.contextService.getWorkspace().folders, this.getFileQueryOptions({
+				cacheKey,
+				maxResults: AnythingQuickAccessProvider.MAX_RESULTS,
```

Comment:
@roblourens I was seeing `maxResults` set to 0 which ended up overloading the quick picker.

Reply:
reverted this to go with a more local approach in the new file system provider